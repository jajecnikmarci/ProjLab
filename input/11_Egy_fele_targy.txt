starttest
add room room1 3
add item room1 FFP2 ffp2
add item room1 Transistor transistor1
add item room1 Transistor transistor2
add item room1 Transistor transistor3
add item room1 Glass glass1
add item room1 Glass glass2
add person room1 Student student1
add person room1 Student student2
printstate room1
printstate student1
pickup student1
pickup student1
printstate room1
printstate student1
printstate student2
pickup student2
pickup student2
pickup student2
pickup student2
printstate room1
printstate student2
endtest