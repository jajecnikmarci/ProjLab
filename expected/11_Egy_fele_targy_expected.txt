room1
	- capacity: 3
	- people: 
		student1
		student2
	- doors: 
	- items: 
		ffp2
		transistor1
		transistor2
		transistor3
		glass1
		glass2
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: false
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room1
	- capacity: 3
	- people: 
		student1
		student2
	- doors: 
	- items: 
		ffp2
		transistor1
		transistor2
		transistor3
		glass1
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: false
	- inventory: 
		glass2
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
student2
	- location: room1
	- stunned: false
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room1
	- capacity: 3
	- people: 
		student1
		student2
	- doors: 
	- items: 
		ffp2
		transistor1
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student2
	- location: room1
	- stunned: false
	- inventory: 
		glass1
		transistor3
		transistor2
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
